README.md

```diff
@@ -1 +1,14 @@
-# atom-ember-colorizer
+# ember-frost-syntax theme
+
+A short description of your theme.
+
+## Usage
+Change syntax theme to Atom Ember Frost
+
+## Theme:
+* Handlebars:
+	* ![Handlebars](images/handlebars-1.png)
+	* ![Handlebars](images/handlebars-2.png)
+
+## Notes:
+Be sure to download a Babel package that handles grammar for a good Ember-Atom experience
```

Comment:
@sglanzer Gonna rely on a good Babel package to handle a users javascript files atom editor css. Grammers are basically regex that give portions of the text the proper css class